trunk/src/app/srs_app_rtc_source.hpp

```diff
@@ -356,7 +359,13 @@ class SrsRtcFrameBuilder
 private:
     srs_error_t packet_video(SrsRtpPacket* pkt);
     srs_error_t packet_video_key_frame(SrsRtpPacket* pkt);
+
+    srs_error_t packet_sps_pps(SrsRtpPacket* pkt);
     srs_error_t packet_sps_pps(SrsRtpPacket* pkt, SrsSample* sps, SrsSample* pps);
+#ifdef SRS_H265
+    srs_error_t packet_vps_sps_pps(SrsRtpPacket* pkt);
+    srs_error_t packet_vps_sps_pps(SrsRtpPacket* pkt, SrsSample* vps, SrsSample* sps, SrsSample* pps);
```

Comment:
```suggestion
    srs_error_t packet_hevc_sh(SrsRtpPacket* pkt);
    srs_error_t packet_hevc_sh(SrsRtpPacket* pkt, SrsSample* vps, SrsSample* sps, SrsSample* pps);
```
those 2 methods are used to packet sequence header rtmp from hevc rtp, so I think theirs name deserve a better one.
vps|sps|pps can consider the hevc sequence header.

Reply:
Sequence header is a name from FLV. 
1. I think `sequence_header` is better than `sh`.
2. If want to change the name, please also change `packet_sps_pps` for H.264.

Reply:
ok